lib/tic_tac_toe_web/controllers/board_controller.ex

```diff
@@ -0,0 +1,19 @@
+defmodule TicTacToeWeb.BoardController do
+    use TicTacToeWeb, :controller
+  
+    def post_move(conn, _params) do
+        conn  
+        |> put_status(:ok)
+        |> send_resp(200, "ok")
+    end
+
+    def fetch_board_update_request(conn) do
+        response = conn.body_params
+        data = Map.fetch(response, "data")
```

Comment:
Why is your function ending with a value assignment?

Reply:
I will delete that. Thanks!